mongodb/migrations/migrations.go

```diff
@@ -180,13 +181,63 @@ var m004 = migrate.Migration{
 	},
 }
 
+var m005 = migrate.Migration{
+	Version:     5,
+	Description: "rename any bucket archive job_status to status",
+	Up: func(db *mongo.Database) error {
+		log.Info("migrating 005 up")
+		ctx, cancel := context.WithTimeout(context.Background(), migrateTimeout)
+		defer cancel()
+		if _, err := db.Collection("bucketarchives").UpdateMany(ctx, bson.M{}, bson.M{
+			"$rename": bson.M{"archives.current.job_status": "archives.current.status"},
+		}); err != nil {
+			return err
+		}
```

Comment:
As a first step, rename any `archives.current.job_status` to `archives.current.status`. This probably could be wrapped into the aggregation below, but this is simple and I didn't want to complicate the aggregation any more.